common/mariadb-galera/helm/templates/statefulset-mariadb.yaml

```diff
@@ -115,6 +115,31 @@ spec:
           privileged: {{ $.Values.initContainers.cleanoscache.securityContext.privileged }}
           runAsUser: {{ $.Values.initContainers.cleanoscache.securityContext.runAsUser | default 0 | int }}
       {{- end }}
+      - name: fix-mariadb-fs-permissions
+        image: "{{ $.Values.image.os.registry }}/{{ $.Values.image.os.project }}/{{ $.Values.image.os.databasename }}:{{ $.Values.image.os.databaseversion }}-{{ $.Values.image.os.imageversion | int }}"
+        imagePullPolicy: {{ $.Values.image.os.pullPolicy | default "IfNotPresent" | quote }}
+        command:
+        - sh
+        - -c
+        - 'chown -R {{ $.Values.userId.database | default 101 | int }}:{{ $.Values.groupId.database | default 101 | int }} /mnt/*'
+        securityContext:
+          runAsUser: {{ $.Values.initContainers.fixFsPermissions.securityContext.runAsUser | default 0 | int }}
```

Comment:
Please rename `initContainers.fixFsPermissions...` to `initContainers.fixMariaDBFsPermissions...` and please add a `initContainers.fixMariaDBFsPermissions.securityContext.runAsUser` option to the values.yaml similar to [cleanoscache](https://github.com/sapcc/helm-charts/blob/8188e613e78536078c381ad3f4b04daa9c55494b/common/mariadb-galera/helm/values.yaml#L856) and add some documentation similar to [wipeDataAndLog](https://github.com/sapcc/helm-charts/blob/8188e613e78536078c381ad3f4b04daa9c55494b/common/mariadb-galera/helm/values.yaml#L69-L71)

Reply:
done!